### ISSUE TITLE
Html widget/summernote eats line breaks 

### ISSUE THREAD

**Issue by InfernalLV** [odoo/odoo#15665]
Impacted versions:
v10, enterprise

Steps to reproduce:
Go to User Preferences -> Signature (or any other html field) and copy paste content from a plain text editor such as notepad or gedit.

Current behavior:
None of the line breaks will be preserved.

Expected behavior:
Newlines are preserved

**Comment by Yenthe666** [comment]
@nim-odoo and @JKE-be this might be one for you?
This indeed happens:
![image](https://cloud.githubusercontent.com/assets/6352350/23938365/5e58cc38-095c-11e7-8008-12e93b6a9a39.png)

**Comment by InfernalLV** [comment]
For what its worth when I looked into it I found out that its like that by design.  
../odoo/addons/web_editor/static/src/js/summernote.js
```
2312-// override summernote clipboard functionality
2313-eventHandler.modules.clipboard.attach = function(layoutInfo) {
2314-    var $editable = layoutInfo.editable();
2315:    $editable.on('paste', function(e) {
2316-        e.preventDefault();
2317-        $editable.data('NoteHistory').recordUndo($editable);
2318:        var pastedText = ((e.originalEvent || e).clipboardData || window.clipboardData).getData('Text');
2319:        var formattedText = pastedText.replace(/([^.!?:;])\r?\n/g, "$1").trim(); // Remove linebreaks which are not at the end of a sentence
2320-        document.execCommand("insertText", false, formattedText);
2321-    });
2322-};
```

**Comment by InfernalLV** [comment]
Its a feature not a bug? :)
This problem is still present in v11

**Comment by Yenthe666** [comment]
I've totally misread your message. I thought you had an official response saying it was desired behaviour. Sorry!

**Comment by JKE-be** [comment]
Hi @Yenthe666 
Yes, as you can read in the code, it is not a bug, but a feature asked ... (bad heuristic for copy/past pdf that adds break-line at each line.) 

But after some discussion, we could maybe revert it. In the hands of @qsm-odoo

**Comment by qsm-odoo** [comment]
In fact, the behavior has been improved in 11.0 (so that the reported case here will work in 11.0) but this will never succeed to handle all cases. We will consider reverting the feature, copy/paste from a PDF may not be that important or should be handled another way.

**Comment by qsm-odoo** [comment]
The "feature" has been reverted with commit https://github.com/odoo/odoo/commit/b1b13dbeede54d3473a0b80e6a8dc21bedfdc5c9, your bug should thus be solved in 10.0.
